File: CS/ThemeFriendlyNameConverter.cs
using System;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Data;

namespace ColosSchemesDemo1 {
    public class ThemeFriendlyNameConverter : IValueConverter {
        public object Convert(object value, Type targetType, object parameter, string language) {
            switch ((ElementTheme)value) {
                case ElementTheme.Light:
                    return "Light";
                case ElementTheme.Default:
                case ElementTheme.Dark:
                    return "Dark";
            }

            return string.Empty;
        }
        public object ConvertBack(object value, Type targetType, object parameter, string language) {
            throw new NotImplementedException();
        }
    }
}

File: VB/ThemeFriendlyNameConverter.vb
Public Class ThemeFriendlyNameConverter
	Implements IValueConverter
	Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
		Select Case DirectCast(value, ElementTheme)
			Case ElementTheme.Light
				Return "Light"
			Case ElementTheme.[Default], ElementTheme.Dark
				Return "Dark"
		End Select

		Return String.Empty
	End Function
	Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
		Throw New NotImplementedException()
	End Function
End Class